### ISSUE TITLE
Remove Overviews from Zero to Robot

### ISSUE THREAD

**Issue by jasondaming** [wpilibsuite/frc-docs#962]
I believe both the [Control System Overview](https://docs.wpilib.org/en/latest/docs/zero-to-robot/step-1/control-system-hardware.html) and the [Software Component Overview](https://docs.wpilib.org/en/latest/docs/zero-to-robot/step-2/control-system-software.html) should be moved from the Zero to Robot section to under Hardware and under Software Tools respectively.  While these are good overview of a bunch of different components many of them are not essential to the Zero to Robot mission.  I do believe we should keep the documents and possibly link to them.

I don't mind doing this I just want to make sure there is agreement before starting on it.

**Comment by Daltz333** [comment]
I believe @Kevin-OConnor rejected this in the past.

**Comment by Kevin-OConnor** [comment]
I'm certainly concerned about doing it as the most common feedback we get in the booth at Champs is "I wish WPILib had a thing that did..." followed by a description of one of the existing WPILib tools. Having said that, additional feedback that it is overwhelming and distracting is certainly compelling. Maybe we should re-look at moving them, but providing an explicit reference/link as part of a Next Steps type thing at the end of the Zero to Robot?

**Comment by jasondaming** [comment]
I 110% agree that an overview is needed.  

If we are going to have a section about 0 to Robot the user is not going to care about looking through a 22 entry long hardware overview or a 14 entry long software overview they want to follow a simple quick step by step guide to get them exactly to a running robot.  That isn't to say they shouldn't read everything in more depth later but making them drink out of the firehose before they know anything or have anything installed is overwhelming.

I have a bunch of other stuff typed up on how I think we can improve the effectiveness of the overview but I will create a new issue for that there.  This issue comes down to should we direct them look at a massive overview that contains LOTS of unneeded information before they do anything in a Zero to Robot section, and I think it is absolutely not.

**Comment by jasondaming** [comment]
The consensus on slack was to move these to Hardware and Software respectively and link to them from the Next Steps / Learn More part of Zero to Robot along with the changes mentioned in #967

**Comment by jasondaming** [comment]
We agreed to do this but will handle it in #967 and #976
